src/modules/UI/scenes/TransactionList/action.js

```diff
@@ -34,8 +34,20 @@ export const fetchTransactions = (walletId: string, currencyCode: string, option
   const wallet = CORE_SELECTORS.getWallet(state, walletId)
   if (wallet) {
     WALLET_API.getTransactions(wallet, currencyCode, options).then(transactions => {
-      const newGroupedTransactionsByDate = groupTransactionsByDate(transactions)
-      dispatch(updateTransactions(transactions, newGroupedTransactionsByDate))
+      let key = -1
+      const transactionsWithKeys = transactions.map((tx) => {
+        const txDate = new Date(tx.date * 1000)
+        const dateString = txDate.toLocaleDateString('en-US', { month: 'short', day: '2-digit', year: 'numeric' })
+        const time = txDate.toLocaleTimeString('en-US', { hour: 'numeric', minute: 'numeric' })
```

Comment:
If we are doing local times string and date string sould we have the en-US hard coded. If so , can you put a TODO comment to localize later.

Reply:
Ok

Reply:
https://app.asana.com/0/361770107085503/630229446509188/f